### PR TITLE
chore: add network timeout in yarnrc

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**
Adds latency to yarnrc

**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
N/A
**Summary**
https://github.com/webpack/webpack-cli/pull/1791 fails due latency

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+ network-timeout 600000
```
Comment (evilebottnawi) [inline .yarnrc:1]: Let's do it less, 600000 ms is very big and potential can freeze CI on long time

Comment (evenstensberg) [reply .yarnrc:1]: what do you suggest? ;)

Comment (evenstensberg) [reply .yarnrc:1]: https://github.com/yarnpkg/yarn/issues/5540#issuecomment-374069461

Comment (evilebottnawi) [reply .yarnrc:1]: hm, okay, sounds good

**Review by evilebottnawi — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Pull request merged**